src/main/java/com/josdem/vetlog/service/VaccinationService.java

```diff
@@ -14,14 +14,56 @@
 limitations under the License.
  */
 
-package com.josdem.vetlog.service;
+ package com.josdem.vetlog.service;
 
 import com.josdem.vetlog.model.Pet;
 import com.josdem.vetlog.model.Vaccination;
 import java.util.List;
 
 public interface VaccinationService {
-    void save(Pet pet);
 
+    /**
```

Comment:
We normaly call this overdesign,  you do not need to add code beyond the acceptance criteria.